source/common/network/utility.cc

```diff
@@ -129,7 +129,11 @@ uint32_t Utility::portFromTcpUrl(const std::string& url) {
   }
 }
 
-Address::InstanceConstSharedPtr Utility::getLocalAddress() {
+// TODO(hennna): Currently getLocalAddress does not support choosing between
+// multiple interfaces and addresses not returned by getifaddrs. In additon,
+// the default is to return a loopback address of type version. This fucntion may
+// need to be updated in the future.
+Address::InstanceConstSharedPtr Utility::getLocalAddress(const Address::IpVersion version) {
```

Comment:
IIRC server.cc (or somewhere) has error checking that this function returns a null address, which can't happen anymore. Can you fix the callers to remove error checking that can't happen?